packages/dev-server/src/middleware/devServerMiddleware.ts

```diff
@@ -57,7 +59,7 @@ export function createDevServerMiddleware({
   middleware.use(remoteDevtoolsCorsMiddleware);
   prependMiddleware(middleware, suppressRemoteDebuggingErrorMiddleware);
 
-  middleware.use(bodyParser.json());
+  middleware.use(bodyParser.json({ limit: bodyParserLimit }));
```

Comment:
Maybe we should just support an environment variable rather than adding a new object to the config spec.

Reply:
Assuming the original motivation for this PR is still relevant, why don't we just pick a higher default e.g. 2 MiB?